### DISCUSSION TITLE
Upload extra files in blobs

### DISCUSSION THREAD

**Discussion by taglme** [goreleaser/goreleaser#1511] [Ideas, Issue Triage, and General discussions]
Is it possible to upload extra files in blob pipe?
We need to upload extra files along with archives to blob provider.
There is "extra_files" option in release pipe. 
Is it possible to use this feature in blob pipe too?

**Comment by caarlos0** [comment]
its not possible on the blobs pipe yet, no... good feature request though I think :)

**Reply by caarlos0** [reply]
https://github.com/goreleaser/goreleaser/issues/1512
